pkg/asset/machines/worker.go

```diff
@@ -579,6 +586,22 @@ func (w *Worker) Generate(dependencies asset.Parents) error {
 			for _, set := range sets {
 				machineSets = append(machineSets, set)
 			}
+
+			// If static IPs are configured, we must generate worker machines and scale the machinesets to 0.
+			if ic.Platform.VSphere.Hosts != nil {
+				logrus.Debug("Generating worker machines with static IPs.")
+				templateName := clusterID.InfraID + "-rhcos"
```

Comment:
@rvanderp3 for template since now that is a provided field...

https://github.com/openshift/installer/pull/6995/files#diff-a13f70d3a1cdbc178348f1d69e11aefee254d13faac647543cb7e4c3d73e51f0R58

Reply:
I believe this is `osImage` in the Machines function which forms the basis of the of the default for `osImageForZone` if `failureDomain.Topology.Template` is empty so I'm not sure this should change.  wdyt?